src/common/pony/detail/atomics.h

```diff
@@ -8,10 +8,12 @@
 #  error "Unsupported platform"
 #endif
 
-#ifndef __cplusplus
+ #ifndef __cplusplus
 #  include <stdalign.h>
 #endif
 
+#include <platform.h>
```

Comment:
In fact I don't even think we need to include `features.h` anymore. This was needed for the `__GNUC_PREREQ` macro but we stopped using it in the Alpine port.